lighthouse-core/config/default.js

```diff
@@ -22,15 +22,16 @@ module.exports = {
     "useThrottling": false,
     "gatherers": [
       "service-worker",
-      "offline"
+      "offline",
     ]
   },
   {
     "passName": "redirectPass",
     "useThrottling": false,
     "gatherers": [
       "http-redirect",
-      "html-without-javascript"
+      "html-without-javascript",
+      "start-url",
```

Comment:
putting it in this pass seems like it could be problematic if this site doesn't do http->https redirect since the base url would then be the http version